src/pages/HomePage.jsx

```diff
@@ -10,77 +10,34 @@ import 'react-toastify/dist/ReactToastify.css';
 function HomePage() {
   const { theme } = useContext(ThemeContext);
 
-  const notify = () =>
+  const notify = () => {
     toast('Want to add your project? Check out the docs!', {
       position: 'top-right',
-      autoClose: 5000,
+      autoClose: 3000,
       hideProgressBar: false,
       closeOnClick: true,
       pauseOnHover: true,
       draggable: false,
       progress: undefined,
-      theme: 'light',
+      theme: theme.mode,
```

Comment:
revert the change. we added only light bg  
```suggestion
      theme: 'light,
```

Reply:
sure, I will switch that back. I thought it looks cool